sponsors.html

```diff
@@ -29,6 +29,8 @@ <h2>FAQ</h2>
           developers to work full-time for a month or longer, accelerating
           projects like Lua stdlib, treesitter parser engine, LSP framework,
           extended marks, embedded terminal, job control, RPC API, and remote UIs.
+          Sponsorship is also used to maintain ongoing infrastructure costs for the
+          Neovim project, such as hosting fees.  
```

Comment:
there are trailing spaces, my OCD kicked in and also as I know nothing about the web I worry it can affect the display xD

Reply:
Fixed, can you squash changes before merging.